backend/Dockerfile

```diff
@@ -4,6 +4,6 @@ FROM registry.suse.com/bci/bci-micro
 ARG DIST_BINARY=src/jetstream/jetstream
 COPY ${DIST_BINARY} /epinio-ui
 
-COPY ui /ui
+COPY backend/ui /ui
```

Comment:
i'm intrigued by this, i would have thought `backend` isn't needed as docker brings in the source folder (`backend`.. which contains `ui`)... but is it now needed as this comes from gorelease's `extra_files` (which needs backend as that's where the new files are)?

Reply:
It looked like `docker buildx` had its context set to the root of that repository, so it couldn't find `ui/` without that change.